report/04_implementation/01_flutter_application.md

```diff
@@ -6,23 +6,149 @@ web application) with a single code base.
 Because of this, the development does not take place platform-specifically,
 but in one programming language and environment.
 
-The programming language used with *Flutter* is *dart*. 
+The programming language used with *Flutter* is *Dart*. 
 *Dart* is an object oriented programming language that is very similar to C#
 or Java. 
 
-// TODO: Mehr Informationen zu DART / Flutter
-
+It's advantage over Java, C# etc. is its modern implementation where it fixes 
+common issues of old languages like Java. Dart for example implements null 
```

Comment:
I'd rather omit this suggestion since elder or elderly is mostly used when referring to old people or antique objects. Imo Java is not yet antique but it isn't quite far away from becoming so 😁

Reply:
If the name doesn't fit you, that's okay. I know elder as a more careful way of expressing something "old", but I could be wrong about that. The repetition of "Java" bothered me more than the old ...

Reply:
Alright, then - I see.
In my next commit I rewrote the whole sentence, so that it no longer states `Java` twice. 👍🏿